Assets/Scripts/Player/PlayerController.cs

```diff
@@ -141,17 +149,31 @@ void Awake()
     void FixedUpdate()
     {
         // Player Movement
-        if (isMoving)
+        if (isMoving == true && _isKinemat == true)
```

Comment:
This section is really verbose, things like == true are unecessary. You're also checking _isKinemat (should probably spell out the full word) every fixed update, when you could just do rb.isKinematic = _isKinemat.

Reply:
took out == parts except for false parts